pytest_fauxfactory.py

```diff
@@ -15,18 +20,42 @@
 )
 
 
-def gen_string(items=None, str_type=None, *args, **kwargs):
-    """Generate a new string type."""
-    item = 0
+class GenString(object):
+    """fauxfactory string generator helper"""
+    __slots__ = ['str_type', 'args', 'kwargs']
+
+    def __init__(self, str_type, *args, **kwargs):
+        if str_type is None:
```

Comment:
How about a test to show that when `str_type is None` that the value associated with it is `in STRING_TYPES`?